Dockerfile

```diff
@@ -1,25 +1,28 @@
 # Copyright (c) 2022 LG Electronics Inc.
 # SPDX-License-Identifier: Apache-2.0
-FROM	ubuntu:20.04
+FROM amd64/ubuntu:20.04
 
-RUN 	apt-get update && apt-get install sudo -y
+RUN	apt-get update && apt-get install sudo -y
 RUN	ln -sf /bin/bash /bin/sh
 
-COPY	. /app
+COPY . /app
 WORKDIR	/app
 
-ENV 	DEBIAN_FRONTEND=noninteractive
+ENV	DEBIAN_FRONTEND=noninteractive
```

Comment:
`noninteractive` mode is useful when we need zero interaction while installing or upgrading the system via apt. However, setting this environment variable to noninteractive has an adverse effect. Because it is inherited by all images and containers built from the image, effectively changing their behavior.

```suggestion
```

Let's look for an alternative.